docs/user-guide/backwards-compatibility.md

```diff
@@ -0,0 +1,12 @@
+# Backwards Compatibility
+
+Balancing the trade-offs of improving a tool and the frustration these changes can cause is a difficult task. One key area in which this affects our users is in the replacement of one rule by another.
```

Comment:
Not just replacement - this is really about rule removal regardless of the reason.